### DISCUSSION TITLE
Port Forwarding

### DISCUSSION THREAD

**Discussion by florianleclerc** [RaspAP/raspap-webgui#1126] [Q&A] [answered]
Hi All, 
I just have a quick question. I'm newbie on RaspAp. I would like to know if it's possible to forward some port ? Or to find an other way to forward port ?Thanks in advance

**Comment by Didr0** [comment, marked as answer]
You can use ngrok or maybe you can set a free VPN with tailscale like i did

**Reply by florianleclerc** [reply]
Perfect, I'll try, many thanks for your help

**Reply by Didr0** [reply]
you're welcome
